x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/teams/teams.tsx

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { lazy } from 'react';
+import { i18n } from '@kbn/i18n';
+import { ActionTypeModel, ValidationResult } from '../../../../types';
+import { TeamsActionParams, TeamsSecrets, TeamsActionConnector } from '../types';
+import { isValidUrl } from '../../../lib/value_validators';
+
+export function getActionType(): ActionTypeModel<unknown, TeamsSecrets, TeamsActionParams> {
+  return {
+    id: '.teams',
+    iconClass: 'logoWindows',
```

Comment:
I think this should be the Microsoft Teams icon. The icon can be downloaded from https://teams.microsoft.com/favicon.ico and you can follow how ServiceNow uses a custom image for the icon: https://github.com/elastic/kibana/blob/master/x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/servicenow/servicenow.tsx#L10.

Reply:
Done!